Sources/SnapshotTesting/Diffable.swift

```diff
@@ -39,8 +48,8 @@ extension String: Diffable {
     guard fst != snd else { return nil }
 
     let hunks = chunk(diff: diff(
-      fst.split(separator: "\n", omittingEmptySubsequences: false),
-      snd.split(separator: "\n", omittingEmptySubsequences: false)
+      fst.split(separator: "\n", omittingEmptySubsequences: false).map(String.init),
+      snd.split(separator: "\n", omittingEmptySubsequences: false).map(String.init)
```

Comment:
doing this to get back into `String` world from `Substring` world

Reply:
:(